docs/cli/variant_add.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "agenta variant add"
+description: "Add a new variant for an app to the agenta platform"
+---
+
+### To add a new variant:
+
+There are two ways to add a new variant to your app.
+- By using a configuration file
+- By creating it from an existing variant
+
+#### Creating a variant from an existing variant
```

Comment:
I don't understand the motivation behind this command. Does this create a config file too?

Reply:
> I don't understand the motivation behind this command. Does this create a config file too?

@mmabrouk 
The command uses the parameters defined in a valid toml file to create a new variant, and afterwards pulls the configuration from the backend and saves it in that variant config file.

Reply:
Still don't understand. The command is
`agenta variant add --from <variant_name>`

Where is the valid toml file???

Reply:
@mmabrouk 

The `agenta variant add` and `agenta variant add --from_variant <variant_name>` would use the config file of the variant that is selected to be the base variant. 

Only the `agenta variant add --config_file <codebase_name>.<config_name>.toml` explicitly defines the config file to use.

Reply:
`agenta variant add --from app.default`
# checks for existing
# checks that 

`cp app.default.toml app.mynewconfig.toml`
`nano app.mynewconfig.toml`
`agenta add-config app.newconfig.toml`

Reply:
```bash
agenta variant add --from app.default
nano app.mynewconfig.toml
agenta variant add --config_file app.mynewconfig.toml
```